package.json

```diff
@@ -33,6 +33,7 @@
     "@ably/vcdiff-decoder": "1.0.4",
     "@types/crypto-js": "^4.0.1",
     "@types/node": "^15.0.0",
+    "@types/request": "^2.48.7",
```

Comment:
I was curious as to what this dependency does, so I had a quick look at [the npm page for it](https://www.npmjs.com/package/@types/request). That led me to click [the GitHub link in the first sentence](https://github.com/request/request). I see you've created https://github.com/ably/ably-js/issues/701 to handle that deprecation in our backlog, however I think we should perhaps have a conversation (outside of the scope of this pull request) around how we label GitHub issues that represent architectural debt like this. 🤔